data/ecosystem/integrations.yaml

```diff
@@ -201,3 +201,13 @@
   docsUrl: https://github.com/fluentci-io/fluentci-engine#-opentelemetry-tracing
   components: [Rust]
   oss: true
+- name: Azure SDKs
+  url: https://github.com/Azure/azure-sdk-for-net
+  docsUrl: https://azure.github.io/azure-sdk-for-net/
+  components: [HTTP]
+  oss: true
+  - name: MassTransit .NET client
+  url: https://github.com/MassTransit/MassTransit
+  docsUrl: https://masstransit.io/
```

Comment:
this also should point to the part of the documentation that speaks about OpenTelemetry

Reply:
https://masstransit.io/documentation/configuration/observability